app/components/developers/query_component.rb

```diff
@@ -60,6 +60,22 @@ def include_not_interested?
       query.include_not_interested
     end
 
+    def collapse_location?
+      query.countries.empty?
+    end
+
+    def location_accordion_classes
+      collapse_location? ? "hidden" : ""
```

Comment:
I don't like having "hidden" in this file when it already exists in the Stimulus controller markup. Do you think you could find a way to reference the one already in the DOM? Or pass it in to the component somehow?